nservicebus/release-policy.md

```diff
@@ -1,13 +1,12 @@
 ---
-title: Release Policy
+title: How and When to upgrade to a new version of NServiceBus

```

Comment:
@udidahan I'm with @SimonCropp here. `Release policy` seems to be the most correct title to me. Perhaps we can add a blog post titled "Considerations on how and when to upgrade" that can give some stronger guidance and link to this one?